FrameworkSystem/Service/SystemAdministratorHandler.py

```diff
@@ -633,8 +632,7 @@ def export_getComponentDocumentation( self, cType, system, module ):
     except Exception as _e:
       return S_ERROR( 'No documentation was found' )
 
-  types_storeHostInfo = []
-  def export_storeHostInfo( self ):
+  def __storeHostInfo( self ):
```

Comment:
The output from
```
gLogger.error( result['Message'] )
```
is really confusing because it is not obvious where the error message comes from, e.g., the ComponentMonitoring

Also
it should be
```
return result
```
instead of creating a new S_ERROR structure.